plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -199,7 +215,8 @@ fu_plugin_mm_inhibit_device (FuPlugin *plugin, FuDevice *device, GError **error)
 
 	/* only do modem port monitoring using udev if the module is expected
 	 * to reset itself into a fully different layout, e.g. a fastboot device */
-	if (fu_mm_device_get_update_methods (FU_MM_DEVICE (device)) & MM_MODEM_FIRMWARE_UPDATE_METHOD_FASTBOOT) {
+	if (fu_mm_device_get_update_methods (FU_MM_DEVICE (device)) & MM_MODEM_FIRMWARE_UPDATE_METHOD_FASTBOOT ||
+	    fu_mm_device_get_update_methods (FU_MM_DEVICE (device)) & MM_MODEM_FIRMWARE_UPDATE_METHOD_MBIM_QDU) {
```

Comment:
I'm a bit confused. The module was in MBIM mode when managed as a modem, and the module is in MBIM mode during the upgrade process. Why do we need to handle the module as a udev device (which is an additional step required only when the modem "switches" between modem-mode and dowload-mode)?

All the work done in the patch to add support to the udev-device-based modem object doesn't seem to be needed at all.